legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/project/TestProjectStructure.java

```diff
@@ -58,6 +61,8 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static org.finos.legend.sdlc.server.project.ProjectStructure.PROJECT_CONFIG_PATH;
```

Comment:
Can we avoid using a static import for this?

Reply:
yes will do